topics/query-subscription-consumer.yaml

```diff
@@ -0,0 +1,9 @@
+topic: query-subscription-consumer
+description: Metric alert consumer
+schemas:
+  - version: 1
+    compatibility_mode: none
+    type: json
+    resource: query-subscription-consumer.schema.json
+    examples:
+      - query-subscription-consumer/1/
```

Comment:
The query subscription consumer is just one client of this schema, I think it's better to avoid this consumer's name should be written anywhere in the schema.

Reply:
Are you able to provide one example for each topic? ...or is that too much?